src/main/java/controller/RacingGameController.java

```diff
@@ -0,0 +1,74 @@
+package controller;
+
+import domain.Car;
+import service.RacingGameService;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class RacingGameController {
+    RacingGameService racingGameService = new RacingGameService();
+    Scanner scanner = new Scanner(System.in);
+
+    public void racingGame() {
+        String inputNames = inputNames();
+        int gameCount = inputGameCount();
+
+        String[] carNames = splitCarNames(inputNames);
+        for (String carName : carNames) {
+            validateInputName(carName);
+        }
+        List<Car> cars = Car.getInstancesByNames(carNames);
+
+        printEmptyLine();
+
+        racingGameService.playRacingGame(cars, gameCount);
+
+        int LongestDistance = racingGameService.getLongestMoveDistance(cars);
+        racingGameService.printWinners(LongestDistance, cars);
+    }
+
+    private String[] splitCarNames(String names) {
+        return names.split(",");
+    }
+
+    private boolean isIllegalInputName(String inputName) {
+        return inputName == null || inputName.isEmpty() || inputName.equals(" ") || isLongerThan5(inputName);
```

Comment:
```suggestion
        return inputName == null || inputName.trim().isEmpty() || isLongerThan5(inputName);
```

`isEmpty()`와 `equals(" ")`가 중복되어 있는 부분이 있어요. 위의 방법을 사용하면 문자열 앞뒤의 공백을 제거하고 빈 문자열인지 확인할 수 있어서 코드가 더 깔끔해질 거예요.

Reply:
리뷰 감사합니다.

알려주신대로 trim메서드를 활용하여 문자열 양 끝의 공백을 제거하고 빈 문자열인지 확인할 수 있게 수정하였는데요.

감사드립니다. !